omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -89,6 +89,11 @@ public Object upload(MultipartFile file, RequestContext context) throws Response
 		String fileCaption = context.getParameter("fileCaption");
 		String instructions = context.getParameter("instructions");
 		
+		// Verify File Size
+		if (attachmentsContext.getMaxUploadFileSize()<file.getSize()) {
```

Comment:
Spacing out:
```java
if (attachmentsContext.getMaxUploadFileSize() < file.getSize())
```

Reply:
@mks-d I will come up with the necessary changes. Thanks

Reply:
Also as @ridmal pointed out [on JIRA](https://issues.openmrs.org/browse/ATT-22?focusedCommentId=246525&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-246525) `file.getSize()` will be given in bytes, whereas `attachmentsContext.getMaxUploadFileSize()` gets back the setting in megabytes.

So you will need to apply the necessary `1024 * 1024` factor.